components/AddPopover/AddPopover.tsx

```diff
@@ -212,6 +230,34 @@ export const AddPopover: React.FunctionComponent<AddPopoverProps> = ({
           >
             Confirm
           </Button>
+          <Snackbar
+            anchorOrigin={{ vertical: "top", horizontal: "right" }}
+            open={noUserSnackbar}
+            autoHideDuration={3000}
+            onClose={() => setUserSnackbar(false)}
+            message="No User Selected"
+          />
+          <Snackbar
+            anchorOrigin={{ vertical: "top", horizontal: "right" }}
+            open={amountSnackbar}
+            autoHideDuration={3000}
+            onClose={() => setAmountSnackbar(false)}
+            message="Amount chosen too large. Please reduce to below 10000"
+          />
+          <Snackbar
+            anchorOrigin={{ vertical: "top", horizontal: "right" }}
+            open={validAmountSnackbar}
```

Comment:
Instead of using 3 separate Snackbars and booleans, I'd recommend just using one boolean to controls whether or no the snackbar is open and using a useState variable to set the appropriate message.